Ansible/playbook.yml

```diff
@@ -0,0 +1,100 @@
+- name: Install To Do app on a new web server
+  hosts: webservers
+  remote_user: ec2-user
+  vars_prompt:
+    - name: api_key
+      prompt: Enter the Trello API key
+      private: no
+
+    - name: api_token
+      prompt: Enter the Trello token
+      private: no
+    
+    - name: board_id
+      prompt: Enter the Trello board ID
+      private: no
+
+    - name: id_not_started
+      prompt: Enter the Not Started list ID
+      private: no
+
+    - name: id_in_progress
+      prompt: Enter the In Progress list ID
+      private: no
+    
+    - name: id_complete_id
+      prompt: Enter the Complete list ID
+      private: no
+
+  tasks:
+    - name: Install Git
+      ansible.builtin.yum:
+        name: git
+        state: latest
+      become: yes
+
+    - name: Install Python3
+      ansible.builtin.yum:
+        name: python3
+        state: latest
+      become: yes
```

Comment:
If you're going to install Python 3.10 anyway, you can skip this task, right? It's a little misleading to keep it. If its purpose is to avoid an error on the version check, you could change that command to something like:
```
command -v python3 >/dev/null && python3 --version
```

The `command -v python3` returns the python location if it's installed. The 
 `> /dev/null` is to avoid actually printing anything to the terminal.  The `&&` means run the right-hand command if the left-hand command succeeds.